3rdparty/glsl-optimizer/src/glsl/ir_print_glsl_visitor.cpp

```diff
@@ -954,7 +954,7 @@ void ir_print_glsl_visitor::visit(ir_texture *ir)
 	ir->coordinate->accept(this);
 	
 	// lod
-	if (ir->op == ir_txl || ir->op == ir_txf || ir->op == ir_txf_ms)
```

Comment:
We handle the `ir_txf_ms` case a few lines below, so this lead to a duplicate last parameter with `texelFetch(Sampler2DMS, ...)`